app/models/activity.rb

```diff
@@ -0,0 +1,15 @@
+# Activity model for customisation & custom methods
+class Activity < PublicActivity::Activity
+  has_many :notification_styles, inverse_of: :activity, dependent: :destroy
```

Comment:
one Activity record has many kinds of methods( way ) to notify users what happened. I call these methods `notification_styles`. It is better than calling them `notifications`.